TFT/src/User/API/icon_list.inc

```diff
@@ -149,4 +149,3 @@ X_ICON (MESH_EDITOR)
 X_ICON (BRIGHTNESS)
 X_ICON (MESH_VALID)
 X_ICON (CALIBRATION)
-//after changing/adding/removing an icon, change the ICON_FLASH_SIGN in Settings.h
```

Comment:
Pleas restore this line.

Reply:
it is on top of the file as in all the other .inc files

Reply:
I deliberately added the line at the end to let anyone editing the files know it. Most of the time the description at the top is not read by many people and this cause problem for others. These types of comments should be left at the places they are relevant to.